eclipse-collections/src/main/java/org/eclipse/collections/impl/bimap/mutable/SynchronizedMutableBiMap.java

```diff
@@ -389,7 +393,7 @@ public ShortIterable collectShort(ShortFunction<? super V> shortFunction)
     }
 
     /**
-     * @deprecated in 7.1. Use {@link OrderedIterable#zip(Iterable)} instead.
+     * @deprecated in 8.0. Use {@link OrderedIterable#zip(Iterable)} instead.
```

Comment:
Ok I understand now. The problem is that you have two commits in this pull request which aren't independent. The first one sets the deprecation to 7.1, which is where @itohro commented here https://github.com/eclipse/eclipse-collections/pull/92/commits/f1d2938ed7946f72bb35e3a120407b5252fc25fc#diff-1ae117ff066180d4b3d50fbfc12db191R392. The second commit changes it to 8.0. For a clean history, 7.1 should never appear in any commit.